torchtune/utils/_import_guard.py

```diff
@@ -5,11 +5,23 @@
 # LICENSE file in the root directory of this source tree.
 
 import torch
-from torchtune.utils._version import torch_version_ge
+from torchtune.utils._version import (
+    _get_torchao_version,
+    _is_fbcode,
+    _nightly_version_ge,
+    torch_version_ge,
+)
 
 # We can only use flex attention / BlockMask if torch version >= 2.5.0 and GPU is Turing / SM75 and above
 _SUPPORTS_FLEX_ATTENTION = (
     torch_version_ge("2.5.0")
     and torch.cuda.is_available()
     and torch.cuda.get_device_capability() >= (7, 5)
 )
+
+torchao_version = _get_torchao_version()
+
+_NEW_TENSOR_CORE_TILED_LAYOUT_API = not _is_fbcode() and (
```

Comment:
mind adding a quick comment explaining this similar to flex attention above?

Reply:
also nit: can you name this as something that implies a yes/no true/false answer (like `_USE_NEW_TENSOR_CORE_TILED_LAYOUT_API`)